package.json

```diff
@@ -168,12 +165,16 @@
     "validate-npm-package-name": "3.0.0",
     "www-authenticate": "0.6.2",
     "xmldoc": "1.1.2",
-    "yarn": "1.22.4",
     "yawn-yaml": "1.5.0"
   },
   "optionalDependencies": {
     "re2": "1.10.5"
   },
+  "peerDependencies": {
```

Comment:
I'm not sure we need these, because installing Renovate as part of a `package.json` is not part of our supported deployment options. Also, I think `npm` is going to start installing `peerDependencies` in their v7.